pkgs/top-level/python-packages.nix

```diff
@@ -2191,6 +2191,8 @@ in {
 
   flask_wtf = callPackage ../development/python-modules/flask-wtf { };
 
+  flatbuffers = callPackage ../development/python-modules/flatbuffers { inherit (pkgs) flatbuffers; };
```

Comment:
personal preference
```suggestion
  flatbuffers = callPackage ../development/python-modules/flatbuffers {
    inherit (pkgs) flatbuffers;
  };
```

Reply:
Doing a cursory scroll through that file, it appears most imports like this are done in one line - I can change it if you like, but I do want to make sure first.

Reply:
If it's short enough, personally, I would like to keep under 80 characters when possible. Like I said, it's just my preference, but I'm not adamant about it.

Reply:
I'll change it then, not really a big deal either way to me. I think this derivation should be good to go after that.